packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/CellValue.js

```diff
@@ -32,7 +32,7 @@ const CellValue = ({ type, value }) => {
     formattedValue = formatNumber(value, {
       // Should be kept in sync with the corresponding value
       // in the design-system/NumberInput: https://github.com/strapi/design-system/blob/main/packages/strapi-design-system/src/NumberInput/NumberInput.js#L53
-      maximumFractionDigits: 21,
+      maximumFractionDigits: 20,
```

Comment:
Thanks man, I am trying to have the ok to merge also the cleanup in the Strapi repo

Reply:
Ah that's why. I was wondering. What are we waiting for?

Reply:
A double check because I have changed the unlimited value for the lifespan field in the Api token v2 (null -> 0)